docs/build/sdk/v2/selling-assets.md

```diff
@@ -0,0 +1,53 @@
+# Selling Assets
+
+Your beliefs around a market might change during the market life time and in
+this case you might want to sell your previously bought assets.
+
```

Comment:
Selling makes sense if prices increase above the informants' prediction even if their beliefs don't change.